components/TARGET_PSA/TARGET_TFM/COMPONENT_SPE/bl2/include/boot_record.h

```diff
@@ -11,6 +11,8 @@
 #include <stdint.h>
 #include <stddef.h>
 #include <limits.h>
+#include "../ext/mcuboot/bootutil/include/bootutil/image.h"
+#include "../ext/mcuboot/include/flash_map/flash_map.h"
```

Comment:
This include path doesn't seem right.

Reply:
for now we are not building secure bootloader in tree as it is provided as binary.
Can we tweak importer to skip bl2 folder.

Reply:
@alzix could you clarify what you mean? Should this include path stay as-is?

Reply:
You were right about include path is being wrong. The only reason why it does not fail compilation is that this header file is never included. This raises a question why is it here? I suggest tweaking importer configuration file to exclude this header file.